client/src/components/Authors.jsx

```diff
@@ -1,25 +1,25 @@
 const Authors = () => {
   return (
-    <div className="bio">
+    <div>
       <div className="row">
           <h6 className="col-12"><b><u>ABOUT THE CREATORS</u></b></h6>
       </div>
       <div>
         <div className="col-lg-4">
-          <img src="images/farrah_bousetta.png" className="img-fluid bioImages" style={{ height: 200, width: 200}} alt="FARRAH PHOTO HERE"/>
-          <p>Farrah Bousetta is an upcoming professional software engineer with previous experience at Facebook, Google and other prestegious tech companies. She gets stuff done. Her nickname is Feisty Farrah.</p>
+          <img src="images/farrah_bousetta.png" className="img-fluid img-responsive bioImages" style={{ height: 200, width: 200}} alt="FARRAH PHOTO HERE"/>
+          <h6 id="author-des">Farrah Bousetta is an upcoming professional software engineer with previous experience at Facebook, Google and other prestegious tech companies. She gets stuff done. Her nickname is Feisty Farrah.</h6>
```

Comment:
should not have duplicate ids. try to avoid ids altogether and use classes instead